### DISCUSSION TITLE
MySQL client removed

### DISCUSSION THREAD

**Discussion by lucadrb** [Azure/CloudShell#457] [Q&A] [answered]
Why was the MySQL client been removed from Cloud Shell?

**Comment by mbifeld** [comment, marked as answer]
There were CVEs in the MySQL client that were not being patched quickly enough for our security measures.  We replaced it with the MariaDB package instead.  There was a delay from when MySQL was removed and when MariaDB was added, and that was a mistake on our part that we'll avoid if a similar package change occurs in the future.
